ansible/vtestbed.yaml

```diff
@@ -230,4 +230,17 @@
   auto_recover: 'False'
   comment: Tests virtual switch vm
 
-
+- conf-name: vms-kvm-wan-4link
+  group-name: vms6-1
```

Comment:
If you use `vms6-1` as group name, then this testbed cannot coexist on same server with other testbeds using this `vms6-1` group. Please be aware of this limitation.
If it is OK, then that's fine. Otherwise, you may need to consider updating this. Including ptf and vm_base related stuff.

Reply:
Based on internal discussion, there is no plan to deploy multi topo on the same server. If there is a plan in the future, will consider this part. Thanks.